merkato/merkato_config.py

```diff
@@ -25,7 +25,7 @@ def load_config():
 
 def create_config():
     # Create new config
```

Comment:
Configs should get stored in the db as opposed to a file, make an issue for it (or I will after this review)